src/config.ts

```diff
@@ -98,15 +98,29 @@ function interpolateSecrets(config: unknown, secrets: Secrets): GoResult<unknown
   return goSync(() => JSON.parse(interpolatedConfig.replace(ESCAPED_ES_MATCH_REGEXP, '$1')));
 }
 
-const loadConfig = (configPath = './config/config.json') => {
+const replaceInterpolationStrings = (obj: any): any => {
+  if (typeof obj === 'string') {
+    obj = obj.replace(/\${MNEMONIC}/g, 'test test test test test test test test test test test junk');
+    obj = obj.replace(/\${FUNDER_RPC_URL_(.*?)_1}/g, 'https://rpc.example.com');
+  } else if (typeof obj === 'object' && obj !== null) {
+    for (const key in obj) {
+      obj[key] = replaceInterpolationStrings(obj[key]);
+    }
+  }
+  return obj;
+};
```

Comment:
```suggestion
const replaceInterpolationStrings = (obj: any): any => {
  return cloneDeepWith(obj, (value) => {
    if (isString(value)) {
      return value
        .replace(/\${MNEMONIC}/g, 'test test test test test test test test test test test junk')
        .replace(/\${FUNDER_RPC_URL_(.*?)}/g, 'https://rpc.example.com');
    }
  });
};
```
I haven't had enough brain to understand recursion :-D So I'd prefer to use lodash library which is already installed.

Reply:
I'm always in favor of applying the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) 😄 Thanks for posting the code snippet 👏🏻